### PR TITLE
NumberField refactor

### PR DESCRIPTION
**Pull request by adamsmietanka**
I moved the Vuex logic into the NumberField component. The component state was moved one level down into the `wing` property, so it can be moved to a separate vuex module in the future. The props were also cleaned - id is now a value computed from the label.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -169,49 +161,23 @@ export default {
...
-        // eslint-disable-next-line
-          console.error(error);
-      });
+  async created() {
+    const choices = await axios.get("http://localhost:5000/airfoils");
+    const airfoil = await axios.get("http://localhost:5000/airfoil", {
+      params: { airfoil: this.selected_airfoil },
+    });
+    this.airfoils = choices.data;
+    this.profile = airfoil.data;
```
Comment (adamsmietanka) [inline src/views/Home.vue:170]: Async-await is more concise

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,85 +1,68 @@
 <template>
-  <container>
+  <section>
```
Comment (adamsmietanka) [inline src/views/Home.vue:2]: `container` wasn't a valid html tag name

File: src/store/index.js [hunk 3]
```diff
@@ -2,27 +2,29 @@ import { createStore } from "vuex";
...
-    chord_tip: 1,
-    angle: 10,
-    selected_airfoil: "test",
+    wing: {
+      span: 10,
+      chord_fuse: 2,
+      chord_tip: 1,
+      angle: 10,
+      selected_airfoil: "0009",
+    },
```
Comment (adamsmietanka) [inline src/store/index.js:11]: Preparing the state to be extracted to a vuex module

File: src/components/NumberField.vue [hunk 4]
```diff
@@ -17,12 +17,29 @@
...
   },
-  emits: ["update:modelValue"],
+  data() {
+    return {
+      timeout: null,
+    };
+  },
+  methods: {
+    setValue(val) {
+      this.timeout && clearTimeout(this.timeout);
```
Comment (adamsmietanka) [inline src/components/NumberField.vue:32]: When `this.timeout` was set before, it'll be reset.

**Review by tosiaw — APPROVED** [review]

**Pull request merged**